### ISSUE TITLE
Onivim2 crashes on completion suggestion

### ISSUE THREAD

**Issue by bartduisters** [onivim/oni2#2764]
Steps to reproduce:
- Clone [https://github.com/bartduisters/completion-bug](https://github.com/bartduisters/completion-bug)
- Open the directory with Onivim2
- Open the file bunny-jump/src/main.js
- On an empty row, type the letter: M

Result: Onivim2 crashes

Version used:
![2020-11-25-1606331741_screenshot_545x462](https://user-images.githubusercontent.com/43420049/100272222-08236980-2f5b-11eb-8345-3496e5998110.jpg)

The debug log of the moment when it crashed can be found in the repository as well. The last part of the log file has this:
```
[INFO]  [ 4.994s] Oni2.Store.dispatch : dispatch: (LanguageSupport
   (Feature_LanguageSupport.Completion
      (Completion.Provider
         (Completion.Exthost
            CompletionProvider.DetailsError {handle = 32;
              errorMsg =
              "Exthost__Client.ReplyError(\"{\\\"$isError\\\":true,\\\"name\\\":\\\"Error\\\",\\\"message\\\":\\\"<semantic> TypeScript Server Error (3.8.3)\\\\nDebug Failure. Alias targets of default exports must have a parent\\\\nError: Debug Failure. Alias targets of default exports must have a parent\\\\n    at Object.assertDefined (tsserver.js:2077:24)\\\\n    at getDefaultExportInfoWorker (tsserver.js:128014:80)\\\\n    at getDefaultLikeExportInfo (tsserver.js:127970:24)\\\\n    at tsserver.js:127742:35\\\\n    at forEachExternalModule (tsserver.js:128169:21)\\\\n    at getAllReExportingModules (tsserver.js:127737:13)\\\\n    at Object.getImportCompletionAction (tsserver.js:127723:31)\\\\n    at getCompletionEntryCodeActionsAndSourceDisplay (tsserver.js:109912:33)\\\\n    at Object.getCompletionEntryDetails (tsserver.js:109876:30)\\\\n    at Proxy.getCompletionEntryDetails (tsserver.js:135365:35)\\\\n    at tsserver.js:145465:57\\\\n    at Object.mapDefined (tsserver.js:562:30)\\\\n    at IOSession.Session.getCompletionEntryDetails (tsserver.js:145463:33)\\\\n    at Session.handlers.ts.createMapFromTemplate._a.<computed> (tsserver.js:144392:61)\\\\n    at tsserver.js:146003:88\\\\n    at IOSession.Session.executeWithRequestId (tsserver.js:145994:28)\\\\n    at IOSession.Session.executeCommand (tsserver.js:146003:33)\\\\n    at IOSession.Session.onMessage (tsserver.js:146027:35)\\\\n    at Interface.<anonymous> (tsserver.js:147342:27)\\\\n    at Interface.emit (events.js:315:20)\\\\n    at Interface._onLine (readline.js:329:10)\\\\n    at Interface._normalWrite (readline.js:474:12)\\\\n    at Socket.ondata (readline.js:186:10)\\\\n    at Socket.emit (events.js:315:20)\\\\n    at addChunk (_stream_readable.js:295:12)\\\\n    at readableAddChunk (_stream_readable.js:271:9)\\\\n    at Socket.Readable.push (_stream_readable.js:212:10)\\\\n    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)\\\",\\\"stack\\\":\\\"Error: <semantic> TypeScript Server Error (3.8.3)\\\\nDebug Failure. Alias targets of default exports must have a parent\\\\nError: Debug Failure. Alias targets of default exports must have a parent\\\\n    at Object.assertDefined (tsserver.js:2077:24)\\\\n    at getDefaultExportInfoWorker (tsserver.js:128014:80)\\\\n    at getDefaultLikeExportInfo (tsserver.js:127970:24)\\\\n    at tsserver.js:127742:35\\\\n    at forEachExternalModule (tsserver.js:128169:21)\\\\n    at getAllReExportingModules (tsserver.js:127737:13)\\\\n    at Object.getImportCompletionAction (tsserver.js:127723:31)\\\\n    at getCompletionEntryCodeActionsAndSourceDisplay (tsserver.js:109912:33)\\\\n    at Object.getCompletionEntryDetails (tsserver.js:109876:30)\\\\n    at Proxy.getCompletionEntryDetails (tsserver.js:135365:35)\\\\n    at tsserver.js:145465:57\\\\n    at Object.mapDefined (tsserver.js:562:30)\\\\n    at IOSession.Session.getCompletionEntryDetails (tsserver.js:145463:33)\\\\n    at Session.handlers.ts.createMapFromTemplate._a.<computed> (tsserver.js:144392:61)\\\\n    at tsserver.js:146003:88\\\\n    at IOSession.Session.executeWithRequestId (tsserver.js:145994:28)\\\\n    at IOSession.Session.executeCommand (tsserver.js:146003:33)\\\\n    at IOSession.Session.onMessage (tsserver.js:146027:35)\\\\n    at Interface.<anonymous> (tsserver.js:147342:27)\\\\n    at Interface.emit (events.js:315:20)\\\\n    at Interface._onLine (readline.js:329:10)\\\\n    at Interface._normalWrite (readline.js:474:12)\\\\n    at Socket.ondata (readline.js:186:10)\\\\n    at Socket.emit (events.js:315:20)\\\\n    at addChunk (_stream_readable.js:295:12)\\\\n    at readableAddChunk (_stream_readable.js:271:9)\\\\n    at Socket.Readable.push (_stream_readable.js:212:10)\\\\n    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)\\\\n\\\\tat Function.create (/tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/tsServer/serverError.js:19:16)\\\\n\\\\tat ProcessBasedTsServer.dispatchResponse (/tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/tsServer/server.js:143:70)\\\\n\\\\tat ProcessBasedTsServer.dispatchMessage (/tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/tsServer/server.js:85:30)\\\\n\\\\tat /tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/tsServer/server.js:58:45\\\\n\\\\tat Emitter.fire (/tmp/.mount_OnivimcRe7rf/usr/share/node/node_modules/@onivim/vscode-exthost/out/vs/base/common/event.js:456:38)\\\\n\\\\tat Reader.onLengthData (/tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/utils/wireProtocol.js:112:30)\\\\n\\\\tat Socket.<anonymous> (/tmp/.mount_OnivimcRe7rf/usr/bin/extensions/typescript-language-features/out/utils/wireProtocol.js:91:42)\\\\n\\\\tat Socket.emit (events.js:315:20)\\\\n\\\\tat addChunk (_stream_readable.js:295:12)\\\\n\\\\tat readableAddChunk (_stream_readable.js:271:9)\\\\n\\\\tat Socket.Readable.push (_stream_readable.js:212:10)\\\\n\\\\tat Pipe.onStreamRead (internal/stream_base_commons.js:186:23)\\\"}\")"}))))

```

**Comment by bryphe** [comment]
Thanks for logging the issue and all the details, @bartduisters !

This looks like the same issue as #2752 - fixed in #2760 . Fix should be in the next nightly

**Comment by bartduisters** [comment]
Sounds great. I'll make sure to close this on confirmation.

**Comment by bryphe** [comment]
I believe this should be fixed now with #2760 - but please feel free to reopen if that's not the case 👍
